packages/framework/src/Component/AddSentryContextSubscriber.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Component;
+
+use App\Environment;
+use RedisException;
+use Sentry\State\Scope;
+use Shopsys\FrameworkBundle\Component\Domain\Domain;
+use Shopsys\FrameworkBundle\Component\Domain\Exception\NoDomainSelectedException;
+use Shopsys\FrameworkBundle\Component\Localization\DisplayTimeZoneProviderInterface;
+use Shopsys\FrameworkBundle\Component\Maintenance\MaintenanceModeSubscriber;
+use Shopsys\FrameworkBundle\Component\Redis\RedisClientFacade;
+use Shopsys\FrameworkBundle\Model\Customer\User\CurrentCustomerUser;
+use Shopsys\FrameworkBundle\ShopsysFrameworkBundle;
+use Symfony\Component\Console\ConsoleEvents;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\Filesystem\Exception\FileNotFoundException;
+use Symfony\Component\HttpKernel\KernelEvents;
+use function Sentry\configureScope;
+
+class AddSentryContextSubscriber implements EventSubscriberInterface
```

Comment:
Just one more thing... What if you change this subscriber to [Monolog Processor](https://symfony.com/doc/current/logging/processors.html)? Probably Sentry will (with proper configuration) register this processor and you will have that informations in your logs as well as in Sentry

What do you think?

Reply:
It is interesting idea. I would like to push current changes further so it solves the main problem and we can implement it in the other PR.